src/test_core.py

```diff
@@ -98,7 +98,7 @@ def test_template_renders__multiline_value_as_valid_json(self):
         }
         rendered = template.render(config=config)
         as_json = json.loads(rendered)
-        self.assertEquals(config, as_json)
+        self.assertEqual(config, as_json)
```

Comment:
Let's revert this change as it will be done in https://github.com/backstage/mkdocs-techdocs-core/pull/223

Reply:
This was a linter change iirc. It should be fine to leave if both places made it.